ios/Classes/LocationController.swift

```diff
@@ -142,6 +142,19 @@ extension LocationOptions {
             }
         }
 
-        return FLT_SETTINGSLocationComponentSettings.make(withEnabled: enabled, pulsingEnabled: nil, pulsingColor: nil, pulsingMaxRadius: nil, showAccuracyRing: showAccuracyRing, accuracyRingColor: accuracyRingColor, accuracyRingBorderColor: accuracyRingBorderColor, layerAbove: nil, layerBelow: nil, puckBearingEnabled: puckBearingEnabled, puckBearingSource: puckBearingSource, locationPuck: locationPuck)
+        return FLT_SETTINGSLocationComponentSettings.make(
+            withEnabled: enabled,
+            pulsingEnabled: nil,
```

Comment:
Dont remember why wasn't pulsing mapped previously, guess it is not available on iOS at all?

Reply:
Location indicator pulsing is available on iOS - 
https://github.com/mapbox/mapbox-maps-ios/blob/8b25fceb8570b0f9ca829685769396f4e02cd7e0/Sources/MapboxMaps/Location/Puck/PuckType.swift#L65

Reply:
Ah ok, just missed in the past then, good to know.